guides/source/debugging_rails_applications.md

```diff
@@ -337,7 +336,6 @@ Processing by ArticlesController#index as HTML
    10:     respond_to do |format|
    11:       format.html # index.html.erb
    12:       format.json { render json: @articles }
-
 (byebug)
```

Comment:
I meant above output from server start. This is always going to keep in changing.